orttraining/orttraining/core/graph/training_op_defs.cc

```diff
@@ -1167,6 +1167,21 @@ Example 4:
         propagateShapeAndTypeFromFirstInput(ctx);
       });
 
+  ONNX_CONTRIB_OPERATOR_SCHEMA(BroadcastGradientArgs)
+      .SetDomain(kMSDomain)
+      .SinceVersion(1)
+      .SetSupportLevel(OpSchema::SupportType::EXPERIMENTAL)
+      .SetDoc("BroadcastGradientArgs")
```

Comment:
Return the reduction indices for computing gradients of s0 op s1 with broadcast.